src/PaymentMethods/PaymentMethodFactory.php

```diff
@@ -97,12 +98,12 @@ class PaymentMethodFactory
         KBC::class => ['kbcpaymentbutton'],
         Paypal::class => ['paypal'],
         PayPerEmail::class => ['payperemail'],
+        PaymentInitiation::class => ['paymentinitiation'],
         EPS::class => ['eps'],
         Emandates::class => ['emandates'],
         Sofort::class => ['sofort', 'sofortueberweisung'],
         Tinka::class => ['tinka'],
         Marketplaces::class => ['marketplaces'],
-        NoServiceSpecifiedPayment::class => ['noservice'],
```

Comment:
Why is this line removed?

Reply:
It's not necessary if you want to make a payment with no method specified just don't send a method name there is no need to check for the noservice method

Reply:
Yes, but it's more readable for some programmers passing a name to the method.